classes/SuperDirt.sc

```diff
@@ -245,6 +257,12 @@ SuperDirt {
 
 		this.closeNetworkConnection;
 
+		netResponders.add(
+			OSCFunc({ |msg, time, tidalAddr|
+				senderAddr.sendMsg("/handshake_reply", *this.handshakeReplyData)
```

Comment:
Maybe this needs to be `tidalAddr.sendMsg(..` ?

Reply:
Yes I tried and it works with that change.

Reply:
Are you happy with the naming convention?

Reply:
Yes the naming convention works for me.